### DISCUSSION TITLE
How to enter the server using the inventory?

### DISCUSSION THREAD

**Discussion by Yakov49** [ORelio/Minecraft-Console-Client#2407] [Q&A]
![image](https://user-images.githubusercontent.com/122409067/211679709-889bcb05-2810-488d-8cde-fb0319dd7fce.png)
In order to enter the server, I need to right-click on 1 slot (compass).
![image](https://user-images.githubusercontent.com/122409067/211679821-6a96a65f-6389-4b57-899f-00cdbd1d7226.png)
Further, when I clicked RMB on 1 slot, the inventory opens, where I need to click on the 6th slot at the very top, on the watermelon, also RMB, and that's it, I'm on the server. Who knows what commands to enter to perform these actions? I try to enter the following: /useitems, /inventory 1 click 6 right. But its doesn't work, please help)

**Comment by BruceChenQAQ** [comment]
Try using `/inventory 1 click 5 right` or `/inventory 1 click 5 rightclick`(new version).

The slot numbering rules for the containers can be found [here](https://wiki.vg/Inventory#Chest).
